### PR TITLE
Minor python fixes

### PR DESCRIPTION
**Pull request by diegoferigo**

### CODE REVIEW

File: gym_ignition/envs/ignition_env.py [hunk 1]
```diff
@@ -52,6 +52,10 @@ def __init__(self) -> None:
         self.ignenv = factory.make(self.md.getEnvironmentName())
         assert self.ignenv, "Failed to create environment " + self.md.getEnvironmentName()
 
+        # Set the verbosity. Run the script as optimized (-O) to decrease the verbosity.
+        gympp.IgnitionEnvironment.setVerbosity(2)
+        assert(gympp.IgnitionEnvironment.setVerbosity(4) or True)
```
Comment (traversaro) [inline gym_ignition/envs/ignition_env.py]: Not blocking, but for the future consider that this construct may not be really readable for not Python experts. It took me a while to realize that with `-O` the assert is not being executed.

**Comment by diegoferigo** [comment]
> Why ctypes instead of os? Is it faster to load?

It seems more idiomatic to me from what I saw. There's no difference in functionality.

File: gym_ignition/envs/ignition_env.py (hunk 1 above)
Comment (diegoferigo) [reply gym_ignition/envs/ignition_env.py]: Yes, it's a dirty trick but it works fine :sweat_smile:

Comment (diegoferigo) [reply gym_ignition/envs/ignition_env.py]: Now you learnt something new ^^

**Pull request merged**